kolibri/plugins/learn/assets/src/state/prepareLearnApp.js

```diff
@@ -0,0 +1,22 @@
+const { currentUserId } = require('kolibri.coreVue.vuex.getters');
+const { MembershipResource } = require('kolibri').resources;
+
+// prepares state that is used for all pages in 'learn' plugin/app
+// currently, this is only the user's memberships
+function prepareLearnApp(store) {
+  const userId = currentUserId(store.state);
+
+  if (userId === null) return Promise.resolve();
+
+  const membershipPromise = MembershipResource.getCollection({ user_id: userId }).fetch();
```

Comment:
It would go in here:
https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/templates/learn/learn.html

`{% kolibri_bootstrap_collection 'membership' 'MemberShipResource' user_id=user.id %}`